azkaban-web-server/src/main/java/azkaban/webapp/servlet/ProjectManagerServlet.java

```diff
@@ -778,6 +778,11 @@ private void ajaxFetchFlowGraph(final Project project, final HashMap<String, Obj
   private void fillFlowInfo(final Project project, final String flowId,
       final HashMap<String, Object> ret) {
     final Flow flow = project.getFlow(flowId);
+    if (flow == null) {
```

Comment:
can we add a unit test?

Reply:
hmm, it seems to be difficult because this is the servlet and we need to delete project...